activerecord/lib/active_record/associations.rb

```diff
@@ -1825,6 +1825,10 @@ def has_one(name, scope = nil, **options)
         #   Specify this association is a polymorphic association by passing +true+.
         #   Note: If you've enabled the counter cache, then you may want to add the counter cache attribute
         #   to the +attr_readonly+ list in the associated classes (e.g. <tt>class Post; attr_readonly :comments_count; end</tt>).
+        #   Note: Since polymorphic associations rely on storing class names in
+        #   the database, that data must remain synchronized with the class name
+        #   used by the Ruby code. When renaming a class, make sure to update
+        #   the data in the corresponding columns.
```

Comment:
Is "columns" the correct term here? Should it mention "rows" instead?

```suggestion
corresponding rows.
```